seed/analysis_pipelines/tasks.py

```diff
@@ -0,0 +1,14 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author
+"""
+
+from seed.analysis_pipelines.bsyncr import (  # noqa: F401
```

Comment:
I don't like this, but celery requires you put your tasks in a file called `tasks.py`. I thought I'd rather keep the tasks inside the pipeline's file then import them here, but open to thoughts.
An alternative would be to change bsyncr.py into a directory, then have a tasks.py in there. Or have a common tasks.py file all pipelines would use (I don't like this alternative)

Reply:
I like this current set up the best. I'm envisioning that once this gets built out, we'll have a `from seed.analysis_pipelines.<analysis_service> import (...` block for each analysis service.

Reply:
That sounds good to me 👍